target/linux/ramips/image/mt7621.mk

```diff
@@ -983,3 +983,20 @@ define Device/zio_freezio
 	kmod-usb-ledtrig-usbport wpad-basic
 endef
 TARGET_DEVICES += zio_freezio
+
+define Device/zyxel_wap6805
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  UBINIZE_OPTS := -E 5
+  IMAGE_SIZE := $(ralink_default_fw_size_32M)
```

Comment:
Please use accurate size, I do not think these variables exist anymore anyway.

Reply:
Right.  They did when I wrote that.  Just demostrates how long this has rotted in my repo :-)

Reply:
Indeed, must be half a year now ... :-)